Jitsi Install failing apt repo install

### ✅ Have you read and understood the above guidelines?

Yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

Jitsi Meet

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/jitsi-meet.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📈 Which Proxmox version are you on?

9.1.9

### 📝 Provide a clear and concise description of the issue.

During jitsi-meeting install getting error related to the jitsi.sources and having malformed entry.

### 🔄 Steps to reproduce the issue.

Run the bash script.

### ❌ Paste the full error output (if available).

Rebuilding with preserved settings:
  Container ID: 103 → 103
  RAM: 4096 MiB | CPU: 4 cores | Disk: 12 GB
  Network: dhcp | Bridge: vmbr0
  Verbose: enabled

  ✔️   Storage tank480 (Free: 400.8GB  Used: 15.9GB) [Template]
  ✔️   Storage tank (Free: 2.9TB  Used: 784.9GB) [Container]
  ✔️   Storage 'tank' (zfspool) validated
  ✔️   Template storage 'tank480' validated
  ✔️   Template search completed
  ✔️   Template debian-12-standard_12.12-1_amd64.tar.zst [local]
  ✔️   LXC Container 103 was successfully created.
  ✔️   Started LXC Container
  ✔️   Network in LXC is reachable (ping)
  ✔️   Customized LXC Container
  ✔️   Set up Container OS
  ✔️   Network Connected: 192.168.1.171 
  ✔️   IPv4 Internet Connected
  ✖️   IPv6 Internet Not Connected
  ✔️   Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
Hit:1 http://security.debian.org bookworm-security InRelease
Hit:2 http://deb.debian.org/debian bookworm InRelease
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Calculating upgrade... Done
The following packages will be upgraded:
  base-files bash bind9-dnsutils bind9-host bind9-libs distro-info-data inetutils-telnet libcap2 libcap2-bin libgnutls30 libpam-modules libpam-modules-bin
  libpam-runtime libpam0g libssl3 libxml2 login openssh-client openssh-server openssh-sftp-server openssl passwd python3-urllib3 ssh
24 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 12.7 MB of archives.
After this operation, 34.8 kB of additional disk space will be used.
Get:1 http://deb.debian.org/debian bookworm/main amd64 base-files amd64 12.4+deb12u13 [70.9 kB]
Get:2 http://security.debian.org bookworm-security/main amd64 libssl3 amd64 3.0.19-1~deb12u2 [2,032 kB]
Get:3 http://deb.debian.org/debian bookworm/main amd64 bash amd64 5.2.15-2+b10 [1,490 kB]
Get:4 http://deb.debian.org/debian bookworm/main amd64 login amd64 1:4.13+dfsg1-1+deb12u2 [616 kB]
Get:5 http://security.debian.org bookworm-security/main amd64 openssh-sftp-server amd64 1:9.2p1-2+deb12u9 [65.9 kB]
Get:6 http://security.debian.org bookworm-security/main amd64 openssh-server amd64 1:9.2p1-2+deb12u9 [457 kB]
Get:7 http://deb.debian.org/debian bookworm/main amd64 libpam0g amd64 1.5.2-6+deb12u2 [92.1 kB]
Get:8 http://deb.debian.org/debian bookworm/main amd64 libpam-modules-bin amd64 1.5.2-6+deb12u2 [75.7 kB]
Get:9 http://deb.debian.org/debian bookworm/main amd64 libpam-modules amd64 1.5.2-6+deb12u2 [291 kB]
Get:10 http://security.debian.org bookworm-security/main amd64 openssh-client amd64 1:9.2p1-2+deb12u9 [993 kB]
Get:11 http://deb.debian.org/debian bookworm/main amd64 passwd amd64 1:4.13+dfsg1-1+deb12u2 [972 kB]
Get:12 http://security.debian.org bookworm-security/main amd64 libgnutls30 amd64 3.7.9-2+deb12u6 [1,408 kB]
Get:13 http://deb.debian.org/debian bookworm/main amd64 libpam-runtime all 1.5.2-6+deb12u2 [161 kB]
Get:14 http://deb.debian.org/debian bookworm/main amd64 libcap2 amd64 1:2.66-4+deb12u2+b2 [27.5 kB]
Get:15 http://deb.debian.org/debian bookworm/main amd64 libxml2 amd64 2.9.14+dfsg-1.3~deb12u5 [688 kB]
Get:16 http://deb.debian.org/debian bookworm/main amd64 distro-info-data all 0.58+deb12u6 [6,528 B]
Get:17 http://security.debian.org bookworm-security/main amd64 bind9-dnsutils amd64 1:9.18.47-1~deb12u1 [156 kB]
Get:18 http://deb.debian.org/debian bookworm/main amd64 libcap2-bin amd64 1:2.66-4+deb12u2+b2 [35.2 kB]
Get:19 http://security.debian.org bookworm-security/main amd64 bind9-host amd64 1:9.18.47-1~deb12u1 [54.8 kB]
Get:20 http://security.debian.org bookworm-security/main amd64 bind9-libs amd64 1:9.18.47-1~deb12u1 [1,180 kB]
Get:21 http://security.debian.org bookworm-security/main amd64 inetutils-telnet amd64 2:2.4-2+deb12u3 [121 kB]
Get:22 http://security.debian.org bookworm-security/main amd64 openssl amd64 3.0.19-1~deb12u2 [1,435 kB]
Get:23 http://security.debian.org bookworm-security/main amd64 python3-urllib3 all 1.26.12-1+deb12u3 [114 kB]
Get:24 http://security.debian.org bookworm-security/main amd64 ssh all 1:9.2p1-2+deb12u9 [175 kB]
Fetched 12.7 MB in 2s (5,486 kB/s)
apt-listchanges: Reading changelogs...
Preconfiguring packages ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../base-files_12.4+deb12u13_amd64.deb ...
Unpacking base-files (12.4+deb12u13) over (12.4+deb12u12) ...
Setting up base-files (12.4+deb12u13) ...
Installing new version of config file /etc/debian_version ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../bash_5.2.15-2+b10_amd64.deb ...
Unpacking bash (5.2.15-2+b10) over (5.2.15-2+b9) ...
Setting up bash (5.2.15-2+b10) ...
update-alternatives: using /usr/share/man/man7/bash-builtins.7.gz to provide /usr/share/man/man7/builtins.7.gz (builtins.7.gz) in auto mode
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../login_1%3a4.13+dfsg1-1+deb12u2_amd64.deb ...
Unpacking login (1:4.13+dfsg1-1+deb12u2) over (1:4.13+dfsg1-1+deb12u1) ...
Setting up login (1:4.13+dfsg1-1+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../libpam0g_1.5.2-6+deb12u2_amd64.deb ...
Unpacking libpam0g:amd64 (1.5.2-6+deb12u2) over (1.5.2-6+deb12u1) ...
Setting up libpam0g:amd64 (1.5.2-6+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../libpam-modules-bin_1.5.2-6+deb12u2_amd64.deb ...
Unpacking libpam-modules-bin (1.5.2-6+deb12u2) over (1.5.2-6+deb12u1) ...
Setting up libpam-modules-bin (1.5.2-6+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../libpam-modules_1.5.2-6+deb12u2_amd64.deb ...
Unpacking libpam-modules:amd64 (1.5.2-6+deb12u2) over (1.5.2-6+deb12u1) ...
Setting up libpam-modules:amd64 (1.5.2-6+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../passwd_1%3a4.13+dfsg1-1+deb12u2_amd64.deb ...
Unpacking passwd (1:4.13+dfsg1-1+deb12u2) over (1:4.13+dfsg1-1+deb12u1) ...
Setting up passwd (1:4.13+dfsg1-1+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../libssl3_3.0.19-1~deb12u2_amd64.deb ...
Unpacking libssl3:amd64 (3.0.19-1~deb12u2) over (3.0.17-1~deb12u2) ...
Preparing to unpack .../openssh-sftp-server_1%3a9.2p1-2+deb12u9_amd64.deb ...
Unpacking openssh-sftp-server (1:9.2p1-2+deb12u9) over (1:9.2p1-2+deb12u7) ...
Preparing to unpack .../openssh-server_1%3a9.2p1-2+deb12u9_amd64.deb ...
Unpacking openssh-server (1:9.2p1-2+deb12u9) over (1:9.2p1-2+deb12u7) ...
Preparing to unpack .../openssh-client_1%3a9.2p1-2+deb12u9_amd64.deb ...
Unpacking openssh-client (1:9.2p1-2+deb12u9) over (1:9.2p1-2+deb12u7) ...
Preparing to unpack .../libpam-runtime_1.5.2-6+deb12u2_all.deb ...
Unpacking libpam-runtime (1.5.2-6+deb12u2) over (1.5.2-6+deb12u1) ...
Setting up libpam-runtime (1.5.2-6+deb12u2) ...
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../libcap2_1%3a2.66-4+deb12u2+b2_amd64.deb ...
Unpacking libcap2:amd64 (1:2.66-4+deb12u2+b2) over (1:2.66-4+deb12u2) ...
Setting up libcap2:amd64 (1:2.66-4+deb12u2+b2) ...
(Reading database ... 20347 files and directories currently installed.)
Preparing to unpack .../libgnutls30_3.7.9-2+deb12u6_amd64.deb ...
Unpacking libgnutls30:amd64 (3.7.9-2+deb12u6) over (3.7.9-2+deb12u5) ...
Setting up libgnutls30:amd64 (3.7.9-2+deb12u6) ...
(Reading database ... 20347 files and directories currently installed.)
Preparing to unpack .../0-libxml2_2.9.14+dfsg-1.3~deb12u5_amd64.deb ...
Unpacking libxml2:amd64 (2.9.14+dfsg-1.3~deb12u5) over (2.9.14+dfsg-1.3~deb12u4) ...
Preparing to unpack .../1-bind9-dnsutils_1%3a9.18.47-1~deb12u1_amd64.deb ...
Unpacking bind9-dnsutils (1:9.18.47-1~deb12u1) over (1:9.18.33-1~deb12u2) ...
Preparing to unpack .../2-bind9-host_1%3a9.18.47-1~deb12u1_amd64.deb ...
Unpacking bind9-host (1:9.18.47-1~deb12u1) over (1:9.18.33-1~deb12u2) ...
Preparing to unpack .../3-bind9-libs_1%3a9.18.47-1~deb12u1_amd64.deb ...
Unpacking bind9-libs:amd64 (1:9.18.47-1~deb12u1) over (1:9.18.33-1~deb12u2) ...
Preparing to unpack .../4-inetutils-telnet_2%3a2.4-2+deb12u3_amd64.deb ...
Unpacking inetutils-telnet (2:2.4-2+deb12u3) over (2:2.4-2+deb12u1) ...
Preparing to unpack .../5-distro-info-data_0.58+deb12u6_all.deb ...
Unpacking distro-info-data (0.58+deb12u6) over (0.58+deb12u5) ...
Preparing to unpack .../6-libcap2-bin_1%3a2.66-4+deb12u2+b2_amd64.deb ...
Unpacking libcap2-bin (1:2.66-4+deb12u2+b2) over (1:2.66-4+deb12u2) ...
Preparing to unpack .../7-openssl_3.0.19-1~deb12u2_amd64.deb ...
Unpacking openssl (3.0.19-1~deb12u2) over (3.0.17-1~deb12u2) ...
Preparing to unpack .../8-python3-urllib3_1.26.12-1+deb12u3_all.deb ...
Unpacking python3-urllib3 (1.26.12-1+deb12u3) over (1.26.12-1+deb12u1) ...
Preparing to unpack .../9-ssh_1%3a9.2p1-2+deb12u9_all.deb ...
Unpacking ssh (1:9.2p1-2+deb12u9) over (1:9.2p1-2+deb12u7) ...
Setting up distro-info-data (0.58+deb12u6) ...
Setting up libssl3:amd64 (3.0.19-1~deb12u2) ...
Setting up inetutils-telnet (2:2.4-2+deb12u3) ...
Setting up libcap2-bin (1:2.66-4+deb12u2+b2) ...
Setting up python3-urllib3 (1.26.12-1+deb12u3) ...
Setting up openssl (3.0.19-1~deb12u2) ...
Setting up libxml2:amd64 (2.9.14+dfsg-1.3~deb12u5) ...
Setting up bind9-libs:amd64 (1:9.18.47-1~deb12u1) ...
Setting up openssh-client (1:9.2p1-2+deb12u9) ...
Setting up bind9-host (1:9.18.47-1~deb12u1) ...
Setting up openssh-sftp-server (1:9.2p1-2+deb12u9) ...
Setting up openssh-server (1:9.2p1-2+deb12u9) ...
rescue-ssh.target is a disabled or a static unit not running, not starting it.
Could not execute systemctl:  at /usr/bin/deb-systemd-invoke line 145.
Setting up bind9-dnsutils (1:9.18.47-1~deb12u1) ...
Setting up ssh (1:9.2p1-2+deb12u9) ...
Processing triggers for man-db (2.11.2-2) ...
Processing triggers for debianutils (5.7-0.5~deb12u1) ...
Processing triggers for libc-bin (2.36-9+deb12u13) ...
  ✔️   Updated Container OS
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  nginx-common
Suggested packages:
  fcgiwrap nginx-doc
The following NEW packages will be installed:
  nginx nginx-common
0 upgraded, 2 newly installed, 0 to remove and 0 not upgraded.
Need to get 641 kB of archives.
After this operation, 1,696 kB of additional disk space will be used.
Get:1 http://security.debian.org bookworm-security/main amd64 nginx-common all 1.22.1-9+deb12u4 [113 kB]
Get:2 http://security.debian.org bookworm-security/main amd64 nginx amd64 1.22.1-9+deb12u4 [528 kB]
Fetched 641 kB in 0s (12.6 MB/s)
Preconfiguring packages ...
Selecting previously unselected package nginx-common.
(Reading database ... 20351 files and directories currently installed.)
Preparing to unpack .../nginx-common_1.22.1-9+deb12u4_all.deb ...
Unpacking nginx-common (1.22.1-9+deb12u4) ...
Selecting previously unselected package nginx.
Preparing to unpack .../nginx_1.22.1-9+deb12u4_amd64.deb ...
Unpacking nginx (1.22.1-9+deb12u4) ...
Setting up nginx-common (1.22.1-9+deb12u4) ...
Created symlink /etc/systemd/system/multi-user.target.wants/nginx.service → /lib/systemd/system/nginx.service.
Setting up nginx (1.22.1-9+deb12u4) ...
Upgrading binary: nginx.
Processing triggers for man-db (2.11.2-2) ...
  ✔️   Installed Dependencies
E: Malformed entry 1 in sources file /etc/apt/sources.list.d/jitsi.sources (absolute Suite Component)
E: The list of sources could not be read.
  💡   apt update failed after adding repository: jitsi
  ⏳   Installing Jitsi MeetE: Malformed entry 1 in sources file /etc/apt/sources.list.d/jitsi.sources (absolute Suite Component)
E: The list of sources could not be read.
  ✖️   in line 30: exit code 100 (APT: Package manager error (broken packages / dependency problems)): while executing command DEBIAN_FRONTEND=noninteractive  apt install -y jitsi-meet

  --- Last 20 lines of log ---
[2026-04-26 11:27:57] [INFO] Setting up Container OS
[2026-04-26 11:27:57] [OK] Set up Container OS
[2026-04-26 11:27:57] [OK] Network Connected: 192.168.1.171 
[2026-04-26 11:27:58] [OK] IPv4 Internet Connected
[2026-04-26 11:27:58] [ERROR] IPv6 Internet Not Connected
[2026-04-26 11:27:59] [OK] Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
[2026-04-26 11:27:59] [INFO] Updating Container OS
[2026-04-26 11:28:33] [OK] Updated Container OS
[2026-04-26 11:28:33] [INFO] Installing Dependencies
[2026-04-26 11:28:37] [OK] Installed Dependencies
[2026-04-26 11:28:37] [WARN] apt update failed after adding repository: jitsi
[2026-04-26 11:28:37] [INFO] Installing Jitsi Meet
[2026-04-26 11:28:37] [ERROR] in line 30: exit code 100 (APT: Package manager error (broken packages / dependency problems)): while executing command DEBIAN_FRONTEND=noninteractive  apt install -y jitsi-meet
  -----------------------------------

  ✖️   Installation failed in container 103 (exit code: 100)
 📋 Installation log: /tmp/jitsi-meet-103-49674483.log

  Error: APT: Package manager error (broken packages / dependency problems)

### 🖼️ Additional context (optional).

_No response_

I tried to have it repair the apt repo and got this:

  💡   apt-get update failed (unknown), trying alternate mirrors...
   💡   Attempting mirror: ftp.au.debian.org
  💡   Mirror ftp.au.debian.org failed (apt-get update error)
   💡   Attempting mirror: debian.ethz.ch
  💡   Mirror debian.ethz.ch failed (apt-get update error)
   💡   Attempting mirror: ftp.nl.debian.org
  💡   Mirror ftp.nl.debian.org failed (apt-get update error)
   💡   Attempting mirror: ftp.debian.org
  💡   Mirror ftp.debian.org failed (apt-get update error)
   💡   Attempting mirror: ftp.us.debian.org
  💡   Mirror ftp.us.debian.org failed (apt-get update error)
   💡   Attempting mirror: mirrors.ocf.berkeley.edu
  💡   Mirror mirrors.ocf.berkeley.edu failed (apt-get update error)
   💡   Attempting mirror: debian.csail.mit.edu
  💡   Mirror debian.csail.mit.edu failed (apt-get update error)
  💡   Multiple mirrors failed (possible CDN synchronization issue).
  💡   Find Debian mirrors at: https://www.debian.org/mirror/list
  Enter a mirror hostname (or 'skip' to abort): skip
  ✖️   All mirrors failed. Check network or try again later.
  ✖️   in line 417: exit code 1 (General error / Operation not permitted): while executing command return 1

  --- Last 20 lines of log ---
[2026-04-26 11:38:46] [INFO] Updating Container OS
[2026-04-26 11:38:46] [WARN] apt-get update failed (unknown), trying alternate mirrors...
[2026-04-26 11:39:02] Attempting mirror: ftp.au.debian.org
[2026-04-26 11:39:02] [WARN] Mirror ftp.au.debian.org failed (apt-get update error)
[2026-04-26 11:39:02] Attempting mirror: debian.ethz.ch
[2026-04-26 11:39:02] [WARN] Mirror debian.ethz.ch failed (apt-get update error)
[2026-04-26 11:39:02] Attempting mirror: ftp.nl.debian.org
[2026-04-26 11:39:02] [WARN] Mirror ftp.nl.debian.org failed (apt-get update error)
[2026-04-26 11:39:02] Attempting mirror: ftp.debian.org
[2026-04-26 11:39:02] [WARN] Mirror ftp.debian.org failed (apt-get update error)
[2026-04-26 11:39:03] Attempting mirror: ftp.us.debian.org
[2026-04-26 11:39:03] [WARN] Mirror ftp.us.debian.org failed (apt-get update error)
[2026-04-26 11:39:03] Attempting mirror: mirrors.ocf.berkeley.edu
[2026-04-26 11:39:03] [WARN] Mirror mirrors.ocf.berkeley.edu failed (apt-get update error)
[2026-04-26 11:39:03] Attempting mirror: debian.csail.mit.edu
[2026-04-26 11:39:03] [WARN] Mirror debian.csail.mit.edu failed (apt-get update error)
[2026-04-26 11:39:03] [WARN] Multiple mirrors failed (possible CDN synchronization issue).
[2026-04-26 11:39:03] [WARN] Find Debian mirrors at: https://www.debian.org/mirror/list
[2026-04-26 11:43:45] [ERROR] All mirrors failed. Check network or try again later.
[2026-04-26 11:43:45] [ERROR] in line 417: exit code 1 (General error / Operation not permitted): while executing command return 1
  -----------------------------------

  ✖️   Installation still failed after APT repair (exit code: 1)

--- a/misc/tools.func
+++ b/misc/tools.func
@@ -1924,8 +1924,8 @@ setup_deb822_repo() {
     echo "Types: deb"
     echo "URIs: $repo_url"
     echo "Suites: $suite"
-    # Flat repositories (suite="./" or absolute path) must not have Components
-    if [[ "$suite" != "./" && -n "$component" ]]; then
+    # Flat repositories (suite ending with "/" or "./") must not have Components
+    if [[ "$suite" != *"/" && -n "$component" ]]; then
       echo "Components: $component"
     fi
     [[ -n "$architectures" ]] && echo "Architectures: $architectures"

